src/system/data-attributes-product.md

```diff
@@ -10,83 +10,87 @@ The installation used to export this data has the sample data installed, and has
 
 |Attribute|Description|
```

Comment:
in the second table, the Attributes are in code.   do you want to make the first table the same?